terraform/layer1-aws/main.tf

```diff
@@ -1,14 +1,14 @@
 terraform {
-  required_version = "~> 0.14.6"
+  required_version = "~> 0.15.1"
```

Comment:
It'd be better to set a specific version due to using the .terraform-version file for tfenv.

Reply:
I pinned terraform's version 0.15.1